static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -96,6 +96,7 @@ export function InviteBanner({
   if (isEligibleForBanner && showBanner) {
     trackAnalytics('github_invite_banner.viewed', {
       organization,
+      members_shown: missingMembers.users.slice(0, 5).length,
```

Comment:
Why is this indexed from 0 to 5?

Reply:
we only show up to 5 members on the banner

Reply:
Is there some constant we use to determine the number of people shown we can use instead of hardcoding the 5

Reply:
probably. i use slice elsewhere in this file too